components/combobox/index.css

```diff
@@ -0,0 +1,609 @@
+        var(--mod-spectrum-combobox-spacing-inline-end-edge-to-text, var(--spectrum-combobox-spacing-inline-end-edge-to-text)) -
+        var(--mod-combobox-button-inline-offset, var(--spectrum-combobox-button-inline-offset, 0px))
+      );
+    }
+  }
+
+  .spectrum-Combobox &.is-invalid {
+    .spectrum-Combobox-input:not(:disabled) {
+      border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-invalid-default, var(--spectrum-combobox-border-color-invalid-default)));
+
+      &:hover,
+      &:active {
+        border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-invalid-hover, var(--spectrum-combobox-border-color-invalid-hover)));
+      }
+  
+      &.is-focused,
+      &:focus {
+        box-shadow: none;
+        border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-invalid-focus, var(--spectrum-combobox-border-color-invalid-focus)));
+      }
+
+      &.is-focused:hover,
+      &:focus:hover {
+        border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-invalid-focus-hover, var(--spectrum-combobox-border-color-invalid-focus-hover)));
+      }
+    }
+
+    &.is-focused,
+    &:focus-within {
+      .spectrum-Combobox-input:not(:disabled) {
+        box-shadow: none;
+        border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-invalid-focus, var(--spectrum-combobox-border-color-invalid-focus)));
+      }
+
+      &:hover {
+        .spectrum-Combobox-input:not(:disabled) {
+          border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-invalid-focus-hover, var(--spectrum-combobox-border-color-invalid-focus-hover)));
+        }
+      }
+    }
+
+    &.is-keyboardFocused {
+      .spectrum-Combobox-input:not(:disabled) {
+        border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-invalid-key-focus, var(--spectrum-combobox-border-color-invalid-key-focus)));
+      }
+    }
+
+    .spectrum-Textfield-validationIcon {
+      color: var(--highcontrast-combobox-icon-color, var(--mod-combobox-alert-icon-color, var(--spectrum-combobox-alert-icon-color)));
+      inset-inline-end: calc(
+        var(--mod-combobox-spacing-inline-icon-to-button, var(--spectrum-combobox-spacing-inline-icon-to-button)) +
+        var(--mod-combobox-button-width, var(--spectrum-combobox-button-width))
+      );
+    }
+
+    &.spectrum-Textfield--quiet {
+      .spectrum-Textfield-validationIcon {
+        inset-inline-end: var(--mod-combobox-button-width, var(--spectrum-combobox-button-width));
+      }
+    }
+  }
+
+  /* Focus Indicator (Ring) */
+  &.is-keyboardFocused {
+    &::after {
+      pointer-events: none;
+      content: '';
+      position: absolute;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      top: 0;
```

Comment:
These styles are already applied to the `::after` element from the `.spectrum-Textfield` component so you should be good to delete these. Those changes have been merged into main so I would do a rebase and check that first